ether/proto0001.go

```diff
@@ -74,22 +83,49 @@ func (c *Connection) serve0001() {
 
 		if err != nil {
 			c.log.Warn("There has been an error reading pkt", err)
-			c.handleErr(2, 0xff)
+			c.handleErr(3, 0xff)
 
 			continue
 		}
 
 		// should at least have a cons
 		if l < 1 {
 			c.log.Warn("Packet malformed")
-			c.handleErr(2, 0xba)
+			c.handleErr(3, 0xba)
 		}
 
-
 		switch buff[0] {
 		case 0xba:
-			// Asking for a key
-			c.handleErr(2, 0xfe)
+			c.log.Debug("Fetching user", buff[2:])
+			conn := manager.FetchUserById(buff[2:])
+
+			if conn == nil {
+				respBuff := []byte{0xca}
+				respBuff = append(respBuff[:], buff)
+
+				_, err := c.bind.Write(respBuff)
+
+				if err != nil {
+					c.log.Warn("user could not be found")
```

Comment:
Isn't it checking if the response to the client went wrong ? This log probably need to be displayed anyway if we enter `conn == nil`